src/segmantic/seg/monai_unet.py

```diff
@@ -61,13 +66,29 @@ class Net(pl.LightningModule):
     augment_spatial: bool = False
     best_val_dice = 0
     best_val_epoch = 0
+    num_samples: int = 4
+    optimizer: dict = {'optimizer': 'Adam',
+                       'lr': 1e-4,
```

Comment:
are the defaults that your tests showed to perform best?

Reply:
Yes exactly. I wasn't able to exhaustively test this but from the 5 settings I used the default one performed best.
![image](https://user-images.githubusercontent.com/61539759/177362002-ac8d21dd-a745-4a88-ab2c-678241bcc726.png)